Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=835267
Bug-Upstream: https://github.com/google/protobuf/issues/2000
Description: Add support for GNU/Hurd which doesn't define PATH_MAX.
Author: Bas Couwenberg <sebastic@debian.org>
Last-Update: 2018-05-29

--- a/src/google/protobuf/compiler/command_line_interface.cc
+++ b/src/google/protobuf/compiler/command_line_interface.cc
@@ -56,6 +56,9 @@
 #include <ctype.h>
 
 #include <limits.h> //For PATH_MAX
+#if defined(__GNU__)
+#define PATH_MAX 4096
+#endif
 
 #include <memory>
 
